src/commands/github/jobLogs/openGitHubLog.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { EOL } from 'os';
+import { FoldingRange } from 'vscode';
+import { IActionContext, parseError } from 'vscode-azureextensionui';
+import { ext } from '../../../extensionVariables';
+import { JobTreeItem } from '../../../tree/JobTreeItem';
+import { StepTreeItem } from '../../../tree/StepTreeItem';
+import { localize } from '../../../utils/localize';
+import { createFoldingRanges } from './createFoldingRanges';
+import { openGitHubLogContent } from './GitHubLogContentProvider';
+import { LogState, parseGitHubLog } from './parseGitHubLog';
+
+export async function openGitHubLog(context: IActionContext, node?: StepTreeItem): Promise<void> {
```

Comment:
I find displaying an empty log file is a bit weird behavior. Users could mistake this for a bug. I noticed this has only happened for me when a job is skipped. However, I assume jobs that do run are capable of not producing any logs as well.

My ideas for possible solutions:
* Show a notification instead, with text similar to "This job was skipped" or "This job did not output any logs"
* Don't show the "Open GitHub Action Log" context menu item for jobs with no logs.

### Example of empty log file:
<img src="https://user-images.githubusercontent.com/12476526/128270289-49a1abfc-c9e2-4ae6-958f-3dddba16fc0e.png" alt="empty log file" width="400" /> 

### This is what GitHub displays:
<img src="https://user-images.githubusercontent.com/12476526/128270436-40e83503-39d9-4094-bb9c-39f2b48f3cc7.png" alt="github job skipped" width="800"/>

Reply:
I think if we find nothing, I'll write "No logs were found" or something to that effect since there are times where nothing is displayed, but they weren't skipped (like if you view the logs before that step has run)